cmd/mps-control-daemon/mount/mount-shm.go

```diff
@@ -54,11 +74,58 @@ func mountShm(c *cli.Context) error {
 		return fmt.Errorf("error creating directory %v: %w", shmDir, err)
 	}
 
-	//  TODO: What should the size of the shm be
-	mountOptions := []string{"rw", "nosuid", "nodev", "noexec", "relatime", "size=65536k"}
+	sizeArg := fmt.Sprintf("size=%v", opts.getShmSize())
+	mountOptions := []string{"rw", "nosuid", "nodev", "noexec", "relatime", sizeArg}
 	if err := mounter.Mount("shm", shmDir, "tmpfs", mountOptions); err != nil {
 		return fmt.Errorf("error mounting %v as tmpfs: %w", shmDir, err)
 	}
 
 	return nil
 }
+
+// getShmSize returns the default shm size.
+// This reads /proc/meminfo to get the total memory to calculate this.
+func (o *options) getShmSize() (size string) {
```

Comment:
```suggestion
func (o *options) getDefaultShmSize() (size string) {
```

Reply:
It doesn't get the default shm size. It also returns the size if set.

Reply:
Then maybe the comment needs to change?

Reply:
It doesn't get the default shm size. It also returns the size if set. Note that this assumes that we want to allow this to be set. I could remove the interactions with `options` to simplify.

Reply:
Removed the addition of `options`. I will add that once it becomes relevant.